common/source/docs/common-mount-targeting.rst

```diff
@@ -81,12 +79,48 @@ To point the gimbal at a particular location (e.g. lat, lon, alt), on the Data s
 .. image:: ../../../images/mount-mp-point-camera-here.png
     :target: ../_images/mount-mp-point-camera-here.png
 
-MAVLink mount commands can be sent from other sources, such as companion computers. See :ref:`mavlink-gimbal-mount` for a commands list and more information.
+QGC Gimbal Controls
+-------------------
+
+`QGC-4.4 (and higher) <https://github.com/mavlink/qgroundcontrol/releases>`__ supports controlling the gimbal from a Toolbar or by directly clicking on the real-time video.  The vehicle must be using ArduPilot 4.5 (or higher)
+
+The gimbal control toolbar should appear at the top once a gimbal is detected.  Click on the toolbar to display the gimbal control buttons
+
+.. image:: ../../../images/mount-targeting-qgc-toolbar.png
+    :target: ../_images/mount-targeting-qgc-toolbar.png
+
+- "Yaw Lock" / "Yaw Follow" controls whether the gimbal maintains an earth-frame yaw target (aka "lock") or moves with the vehicle's yaw (aka "follow")
+- "Center" centers the gimbal (or more precisely moves it to the roll, pitch and yaw angles held in :ref:`MNT1_NEUTRAL_X<MNT1_NEUTRAL_X>`, :ref:`MNT1_NEUTRAL_Y<MNT1_NEUTRAL_Y>`, :ref:`MNT1_NEUTRAL_Z<MNT1_NEUTRAL_Z>`)
+- "Tilt 90" points the gimbal directly downwards
+- "Point Home" points the gimbal at the home location
+- "Retract" retracts the gimbal (or more precisely moves it to the roll, pitch and yaw angles held in :ref:`MNT1_RETRACT_X<MNT1_RETRACT_X>`, :ref:`MNT1_RETRACT_Y<MNT1_RETRACT_Y>`, :ref:`MNT1_RETRACT_Z<MNT1_RETRACT_Z>`)
+
+The gimbal may also be controlled by clicking on the real-time video screen
+
+- Push the "Setting" button
+- Check "Enable on Screen Camera Control" and set "Control type" to "Click to point"
+- Enter the camera's Horizontal FOV and Vertical FOV
+- Maximise the real-time video and click anywhere on the screen and the gimbal should move appropriately
+
+.. image:: ../../../images/mount-targeting-qgc-clicktopoint.png
+    :target: ../_images/mount-targeting-qgc-clicktopoint.png
+
+To point the gimbal at a Location
+
+- While the vehicle is flying, maximise the map
+- Click anywhere on the map and select "ROI at location"
+
+.. image:: ../../../images/mount-targeting-qgc-roi.png
+    :target: ../_images/mount-targeting-qgc-roi.png
+
```

Comment:
should add how to release ROI lock, point home, and click to point actions....I have not tried but click to point does not track like home and ROI, right? probably should mention that also

Reply:
Hi @Hwurzburg, OK, I've added a line mentioning how to cancel the ROI.  .. also rebased on master